.github/workflows/collector-slim.yml

```diff
@@ -78,53 +70,54 @@ jobs:
-        uses: ./.github/actions/retag-and-push
-        with:
-          src-image: ${{ inputs.collector-image }}
-          dst-image: ${{ inputs.collector-image }}-${{ matrix.arch }}-base
-          username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
-          password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
+            if [[ "${{ contains(github.event.pull_request.labels.*.name, 'address-sanitizer') }}" == "true" ]]; then
+              echo "ADDRESS_SANITIZER=true" >> "$GITHUB_ENV"
+            fi
+          fi
 
-      - name: Push builder to stackrox-io
-        if: github.event_name == 'push' || needs.set-environment.outputs.collector-builder-tag != env.DEFAULT_BUILDER_TAG
+      - name: Create secrets.yml
         run: |
-          docker tag "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}" "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
-          docker push "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
+          {
+            echo "---"
+            echo "stackrox_io_username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}"
+            echo "stackrox_io_password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}"
+            echo "rhacs_eng_username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}"
+            echo "rhacs_eng_password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}"
+          } > ${{ github.workspace }}/ansible/secrets.yml
 
-      - name: Retag and push rhacs-eng -slim
-        uses: ./.github/actions/retag-and-push
-        with:
-          src-image: ${{ inputs.collector-image }}
-          dst-image: ${{ env.RHACS_ENG_IMAGE }}-${{ matrix.arch }}-slim
-          username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
-          password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
-
-      - name: Retag and push rhacs-eng -base
-        uses: ./.github/actions/retag-and-push
-        with:
-          src-image: ${{ inputs.collector-image }}
-          dst-image: ${{ env.RHACS_ENG_IMAGE }}-${{ matrix.arch }}-base
-          username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
-          password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
-
-      - name: Push builder to rhacs-eng
-        if: github.event_name == 'push' || needs.set-environment.outputs.collector-builder-tag != env.DEFAULT_BUILDER_TAG
+      - name: Build images
+        if: |
+          github.event_name == 'push' ||
+          matrix.arch == 'amd64' ||
+          !contains(github.event.pull_request.labels.*.name, 'skip-multiarch-builds')
         run: |
-          docker tag "quay.io/stackrox-io/collector-builder:${COLLECTOR_BUILDER_TAG}" "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
-          docker push "quay.io/rhacs-eng/collector-builder:${COLLECTOR_BUILDER_TAG}-${{ matrix.arch }}"
+          ansible-galaxy collection install community.general
+          ansible-galaxy collection install community.docker
```

Comment:
consider adding these to a `ansible-requirements.yml` file and installing with `ansible-galaxy collection install -r ansible-requirements.yml`